clwb/BUILD

```diff
@@ -83,6 +83,7 @@ java_library(
         "//intellij_platform_sdk:plugin_api",
         "//sdkcompat",
         "//clwb/sdkcompat",
+        "//shared",
```

Comment:
is the order correct here, ie why `clwb` is inbetween?

Reply:
No it's not. I'll format the entire file in a new PR?